### ISSUE TITLE
about the code of speech recognition based on pytorch

### ISSUE THREAD

**Issue by lvyilan23** [qute012/Real-Time-SR#1]
Hi, can you give me the code of speech recognition based on pytorch?

**Comment by dobby-seo** [comment]
Hi, you means python code for training?

2020년 10월 15일 (목) 오후 2:48, lvyilan23 <notifications@github.com>님이 작성:

**Comment by lvyilan23** [comment]
yes. Because I want to know how to set the input and output in android according to the python model.

**Comment by dobby-seo** [comment]
Thank you for attention. that app returns bad recognition, but model works
well not in application for test set. I refered
https://github.com/tugstugi/pytorch-speech-commands.
My code is similar to that, if you wanna get mine, tell me :D.

2020년 10월 15일 (목) 오후 9:26, lvyilan23 <notifications@github.com>님이 작성:

**Comment by lvyilan23** [comment]
Hi, thank u for responding. I still wants to get your code. In fact, I just want to know how the input and output shape correspond in pytorch and android. through your code. The performance is not important.
Thank u very much!!

**Comment by lvyilan23** [comment]
Hi, can u give me a copy of your paper? I used my own trained model in Android, and met some mistakes. I want to know how you train it.

**Comment by dobby-seo** [comment]
Hello, i’m sorry for that i can’t find my code. It’s too old T.T. I can
help you. You may meet error when resnet implements skip connection, right?
It solves not to use any functions for residual.

2020년 10월 19일 (월) 오후 9:29, lvyilan23 <notifications@github.com>님이 작성:

**Comment by dobby-seo** [comment]
And when i used tf-lite l, it’s good works~ so you need to try to make
tf-lite speech command models.

2020년 10월 20일 (화) 오전 11:54, 서쩰루리 <ejrwls012@gmail.com>님이 작성:
